src/util/wrenembed.c

```diff
@@ -114,14 +116,19 @@ int WRENEMBED_encodeAndDump(int argc, char* args[])
   return EXIT_SUCCESS;
 }
 
+void WRENEMBED_usage() {
+  fputs("dome -e | --embed sourceFile [moduleName] [destinationFile]\n", stderr);
```

Comment:
I guess this is only `dome` if it isn't the standalone embed tool.

Reply:
Yes it is only used in the dome function. Was created only to avoid repetition.

Reply:
This should have `[--]` before the `sourceFile`, because it is a legal, but optional addition. The same applies to the DOME usage message.